spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/support/converter/JacksonPubSubMessageConverter.java

```diff
@@ -0,0 +1,71 @@
+/*
+ *  Copyright 2018 original author or authors.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+package org.springframework.cloud.gcp.pubsub.support.converter;
+
+import java.io.IOException;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.google.pubsub.v1.PubsubMessage;
+
+import org.springframework.integration.support.json.JacksonJsonUtils;
+import org.springframework.util.Assert;
+
+/**
+ * A message converter using Jackson JSON.
+ *
+ * @author Chengyuan Zhao
+ */
+public class JacksonPubSubMessageConverter implements PubSubMessageConverter {
+
+	private final ObjectMapper objectMapper;
+
+	/**
+	 * Constructor
+	 * @param trustedPackages the packages trusted for deserialization in addition to the
+	 * default trusted packages listed in {@link JacksonJsonUtils}.
+	 */
+	public JacksonPubSubMessageConverter(String... trustedPackages) {
+		this.objectMapper = JacksonJsonUtils.messagingAwareMapper(trustedPackages);
```

Comment:
One more time: are we agreed to have Spring Integration as a `required` dependency?
Seems for me the story was to have `PubSubTemplate` independent of Spring Integration.

With the required Jackson dependency for the `PubSubTemplate` defaults, we end up with required Spring Integration because of this "fancy" `messagingAwareMapper()` utility. 

Is that really expected?

Thanks

Reply:
I am with having it required. @meltsufin said he was fine with it being required, too.